### ISSUE TITLE
What happened?

### ISSUE THREAD

**Issue by Bacon-Ranch** [twrecked/hass-aarlo#141]
Newbie, this will be obvious I am sure. I installed this last night sucessfully. Today it was showing that all my devices placed in the custom lovelace card were missing. tried several things including uninstall and reinstall. Now I can not find hass-aarlo or aarlo in the HACs directory. I found a similiar arlo file that looks identical, tried it with no sucess. I noticed "somewhere" that it was last updated a day ago, is my timing just really crappy and that is why I had these problems or is t something else? I was about to get rid of my 12 Arlo cameras and had decided that AArlo made them useful, but now I think they need to go, Netgear as a company is dreadful to deal with and these arent cheap.

**Comment by rccoleman** [comment]
There was an update yesterday, and it’s still visible in the Integrations section of HACS for me under Arlo Camera Support. It’s working fine for me, including the Lovelace card.

If you don’t see it in HACS, maybe you’re using an old version of HACS, or maybe you need to hit the three dots in the upper right corner and choose ‘Reload Data’.

**Comment by Bacon-Ranch** [comment]
thanks, I have tried that now, no change. I get a repo from Sherrell with this title:Arlo Camera Support and it looks the same as Twrecked's. I did try it and it was not working either. Weird, it was working great for about 12 hours, then this morning it went pear shaped. Thanks again.

**Comment by twrecked** [comment]
Hi. Sorry you're having issues. Firstly, these are where the components get installed by HACS.

The main component goes here:
`custom_components/aarlo/`

The lovelace component ends up here:
`www/community/lovelace-hass-aarlo/`

Are you ok to look through debug? If you add this to the configuration file it should give some extra debug in the `home-assistant.log` file. It should tell me what's going wrong.

```
logger:
    default: info
    logs:
        pyaarlo: debug
```

**Comment by Bacon-Ranch** [comment]
thank you for replying. I did get this resolved....did end up installing from the user sherrell, seems to be same as yours but your version still does not show up in HACS searches with Hassio, it did the first time. Not sure what the issue was but likely user error in the configuartion or installtion. Thanks again, it has saved my substantial in investment in Arlo infrastructure.  If I recall now, it was YAML formatiing, I copied and pasted your block and had errors because I had all the platforms listed already within my configuartion.yaml and was getting errors. I copied binary_sensor and sensor to the correct area but had pasted sensor in binary_sensor and vice versa.
